auth/authenticator.go

```diff
@@ -106,11 +107,20 @@ func (a *Authenticator) readAccessTokenFile() error {
 	userRecords := make(map[string]User)
 	for _, record := range records {
 		token := record[4]
+
+		// superuser column: interpret "truthy" and "falsey" values as booleans
+		var isSuper bool
+		switch strings.ToLower(record[5]) {
+		case "1", "true":
+			isSuper = true
+		}
```

Comment:
is there a reason to allow this kind of lax specification vs enforcing it being either 1 / 0 or true / false?